.publishrc

```diff
@@ -4,7 +4,7 @@
     "uncommittedChanges": true,
     "untrackedFiles": true,
     "sensitiveData": true,
-    "branch": "master",
+    "branch": /(master|release)/,
```

Comment:
If the condition we want is:

> allow publishing from either the `master` or `release*` branches.

Then this needs to change a bit. Probably something like `/^(master|(release\/\d\.\d))$/` (either `master` or `release/x.y`)?

Reply:
Updated in 5b81295 - tweaked the regex slightly to match patch versions - `release/1.0.0`